.github/workflows/cypress-explorer-e2e.yml

```diff
@@ -79,12 +79,10 @@ jobs:
       ## Setup a Vega wallet for our user
       ######
 
-      - name: Set up Vegawallet
-        uses: ./frontend-monorepo/.github/actions/setup-vegawallet
+      - name: Set up Vegawallet for docker
+        uses: ./frontend-monorepo/.github/actions/setup-vegawallet-docker
         with:
-          recovery: ${{ secrets.TRADING_TEST_VEGA_WALLET_RECOVERY }}
           passphrase: ${{ secrets.CYPRESS_TRADING_TEST_VEGA_WALLET_PASSPHRASE }}
```

Comment:
Non blocking: Since the wallet mnemonic is exposed there's not much point in having the passphrase in a secret. We can include it as a file in the same location as the recovery file and there won't be a need to create it in the workflow

Reply:
Yeah, basically you're right, as when we know mnemonic, hiding password is pointless as you can set it to whatever you want. But just for "security purpose" I don't want to share it here now.